cinema/models.py

```diff
@@ -27,6 +27,7 @@ class Actor(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
 
+    @property
```

Comment:
There is no need to add `property` decorator here, remove it

Reply:
There is no need to `add property` decorator here, remove it
Also better to use `f` string  instead of `+ " " + `